stories/s2l2a.stories.js

```diff
@@ -341,6 +342,55 @@ export const GetMapProcessingEvalscripturlVersion2 = () => {
   return wrapperEl;
 };
 
+export const GetMapProcessingStaging = () => {
+  if (!process.env.CLIENT_ID || !process.env.CLIENT_SECRET) {
```

Comment:
I'm not sure the same client id and secret work for both stage and production. Did you test?

Just want to make sure. FFTR.

Reply:
If I understand correctly it is possible to fake whatever credentials in staging, though I don't know for sure. But we can always change if this is not enough.

Reply:
Ok, I agree. Resolving